src/main/java/org/webjars/taglib/LocateTag.java

```diff
@@ -39,6 +39,15 @@
 	private Iterator<String> result;
 
 	/**
+	 * Whether the path we're looking for do not specify the version number.
```

Comment:
Please fix the grammar issue in this sentence. I believe something like "Whether to emit a path that does not contain version information"